web/src/main/java/org/springframework/security/web/util/matcher/RequestMatchers.java

```diff
@@ -0,0 +1,56 @@
+package org.springframework.security.web.util.matcher;
+
+/**
+ * A factory class to create {@link RequestMatcher} instances.
+ *
+ * @author Christian Schuster
+ * @since 6.1
+ */
+public final class RequestMatchers {
+
+	/**
+	 * Creates a {@link RequestMatcher} that matches if at least one of the
+	 * given {@link RequestMatcher}s matches, if <code>matchers</code>
+	 * are empty then the returned matcher never matches.
+	 * @param matchers the {@link RequestMatcher}s to use
+	 * @return the any-of composed {@link RequestMatcher}
+	 */
+	public static RequestMatcher anyOf(RequestMatcher... matchers) {
```

Comment:
Will you please use `AndRequestMatcher` here? The reason is so that the result also gets `AndRequestMatcher#matcher`.

Reply:
Done.

I had to handle the case with zero matchers explicitly, because the constructor of `AndRequestMatcher` throws an exception in that case.